11-model-comparison/11_model_comparison.ipynb

```diff
@@ -0,0 +1,462 @@
+{
```

Comment:
<p>Strike filler words like 'Well' at the beginning here also. Consider also dialing back the exaggerated shock. Your audience is inexperienced but ideally should not be spoken to like actual children. Including bits like "Did you catch that?" and "mathese."</p><p>Consider also breaking up the text here with either code blocks or pictures. I'm feeling myself skimming when I really shouldn't be. Don't be afraid to typeset an actual equation instead of verbally describing the equation--this ends up being much more confusing.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/TuringLang/TuringTutorials/pull/275/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='14'/>